Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -568,7 +568,7 @@ export class BotProject implements IBotProject {
       outDir,
       metaSchema: undefined,
       allLocales: undefined,
-      templateDirs: [],
+      templateDirs: templateDirs || [],
```

Comment:
Do you mean ?? - I think this will set to a true?

Reply:
it's going to be the value of templateDirs or empty array which the library expects.